config/v2/resolvers.go

```diff
@@ -175,6 +176,24 @@ func ResolveTfLint(commons ...Common) v1.TfLint {
 	}
 }
 
+func ResolveAtlantis(commons ...Common) Atlantis {
+	enabled := false
+	for _, c := range commons {
+		if c.Tools != nil && c.Tools.Atlantis != nil && c.Tools.Atlantis.Enabled != nil {
+			enabled = *c.Tools.Atlantis.Enabled
+		}
+	}
+
+	roleName := lastNonNil(AtlantisRoleNameGetter, commons...)
+	rolePath := lastNonNil(AtlantisRolePathGetter, commons...)
+
+	return Atlantis{
+		Enabled:  &enabled,
```

Comment:
should we ditch Enabled and instead go off the presence/absence of the atlantis block?

Reply:
oh, I see. you need Enabled to properly resolve the inheritance chain

Reply:
I don't think that would work in the case that you turn it on in defaults and want to turn it off somewhere else.